contracts/core/libs/attestations-registry/AttestationsRegistryConfigLogic.sol

```diff
@@ -145,4 +311,62 @@ contract AttestationsRegistryConfigLogic is
     _authorizedRanges[issuer].pop();
     emit IssuerUnauthorized(issuer, firstCollectionId, lastCollectionId);
   }
+
+
+  function _registerAttestationsCollectionTag(
```

Comment:
would replace with setTagForAttestationsCollection, and not check, especially in internal collection